build.sbt

```diff
@@ -82,7 +82,7 @@ assemblyMergeStrategy in assembly := {
 }
 
 // Don't run Integration tests in default run
-testOptions in Test += Tests.Argument("-l", "no.ndla.articleapi.ESIntegrationTest")
+testOptions in Test += Tests.Argument("-l", "no.ndla.articleapi.tag.ESIntegrationTest")
```

Comment:
Ah. Skjønner behovet for egen tag. Men vi vil vel at tagen skal være felles for alle som bruker ES, på tvers av repoer? I så fall ville jeg droppet `articleapi` i navnet, og hatt `no.ndla.tag.ESIntegrationTest`